js/ui/windowManager.js

```diff
@@ -119,6 +119,55 @@ var DisplayChangeDialog = class extends ModalDialog.ModalDialog {
     }
 };
 
+var EOSShellWobbly = class extends EndlessShellFX.Wobbly {
```

Comment:
I think this commit should only contain the C effect implementation itself.
The remaining bits of the window manager implementation should be squashed with a3140aacbad3b9874fa7e8c73f930d11812307df ("windowManager: Integrate wobbly window effect")